kolibri/utils/options.py

```diff
@@ -45,9 +45,25 @@
 HTTP_PORT
 RUN_MODE
 URL_PATH_PREFIX
+    Serve Kolibri from a subpath under the main domain. Used when serving multiple applications from
+    the same origin. This option is not heavily tested, but is provided for user convenience.
 LANGUAGES
-ZIP_CONTENT_HOST
+ZIP_CONTENT_URL_PATH_PREFIX
+    The zip content equivalent of URL_PATH_PREFIX - allows all zip content URLs to be prefixed with
+    a fixed path.
```

Comment:
It wasn't clear to me from reading this whether it was affecting how it was being served, or how the frontend was pointing to it, or both. From reading the code, I think it's maybe both? Which is probably OK, since there's not much of a use case for having a different prefix on the other server than locally (seeing as if the origin is changed, presumably the local zip server wouldn't even be used). But it might be good to be clearer about how it's used.

Reply:
It's both, yes - although in the case where you've set the origin option too, the fact that it is now serving from the other endpoint is a moot point.